src/System.Collections/tests/System.Collections.Tests.csproj

```diff
@@ -124,6 +124,7 @@
     <Compile Include="Generic\List\List.Generic.Tests.AddRange.cs" />
     <Compile Include="Generic\List\List.Generic.Tests.BinarySearch.cs" />
     <Compile Include="Generic\List\List.Generic.Tests.Constructor.cs" />
+    <Compile Include="Generic\List\List.Generic.Tests.ConvertAll.cs" />
```

Comment:
><Compile Include="Generic\List\List.Generic.Tests.ConvertAll.cs" /> [](start = 4, length = 67)

Just as an FYI, I will have to change this to be only added when `'$(TargetGroup)'=='netstandard1.7'` once I do the System.Collections update, which will cause this tests to not be runned until we add support in buildtools for executing tests for netcoreapp1.1